pages/tag_selector.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Tag Selector
+authors:
+  - Mitchell Krome
+keywords: Tagging
+edited: 4 January 2017
+---
+
+The tag selector is opened from the view menu.
+It allows the selection of objects based on their tags.
+
+![ ](images/tag_selector.png)
+
+# Editor Layout
+
+Each row in the editor represents a new operation in the query.
+The columns define the required elements of the operation.
+
+## Relation
+
+The relation column specifies a logical operator describing how it is related to the row above.
+The first row therefore has no relation.
+There are only two possible relations: **and** or **or**.
+
+Precedence of the relations are setup such that the query in the figure above would be interpreted as:
+
+```
+("highway" is "residential" and "source" contains "bing") or ("highway" is "primary" and "source" contains "bing")
+```
+
+That is, **and** relations take precedence over **or** relations. Within the editor the **and** relation is indented slightly to show this precedence.
+
+## Key
+
+The key is the tag which this operation is looking for.
+The behavior when the object does not have a given key is dependent on which comparison is being made.
```

Comment:
I would remove this sentence. It is expressed more precisely in the Comparison section.